src/js/mep-player.js

```diff
@@ -1,3 +1,19 @@
+/**
+ * Returns true if targetNode appears after sourceNode in the dom.
+ * @param {HTMLElement} sourceNode - the source node for comparison
+ * @param {HTMLElement} targetNode - the node to compare against sourceNode
+ */
+function isAfter(sourceNode, targetNode) {
+	if (!sourceNode) {
+		throw new TypeError('Source node is missing or undefined');
+	}
+
+	return !!(
+		targetNode &&
+		sourceNode.compareDocumentPosition(targetNode) & Node.DOCUMENT_POSITION_PRECEDING
```

Comment:
Node.DOCUMENT_POSITION_PRECEDING is the concern for old browsers since I have never seen that value before